frontend/src/app/app.component.spec.ts

```diff
@@ -15,6 +15,8 @@ describe('AppComponent', () => {
     TestBed.configureTestingModule({
       declarations: [
         AppComponent,
+        FooterComponent,
```

Comment:
`FooterComponent` and `HeaderComponent` are not imported here TypeScript will not know what this symbols mean in this file.

See: https://www.typescriptlang.org/docs/handbook/modules.html